### DISCUSSION TITLE
Open discussion: Ideas, suggestions, small fixes.

### DISCUSSION THREAD

**Discussion by nitotm** [nitotm/efficient-language-detector#4] [General]
Share anything you think could be improved, also for small errors that are not worth an `Issue`.

**Comment by eurosat7** [comment]
I offered a pull request with a complete overhaul adding some more architecture and tooling to this project. It might be a bit overwhelming but it will make things in the future easier should you ever consider to release it into the wild.

**Comment by bobitza** [comment]
How i train with my dataset ?

**Comment by nitotm** [comment]
> How i train with my dataset ?

The trainer is not released. 
I could release it in the future, however, at present, the limited time I devote to ELD I put it into developing the new version and upgrading the various ELD repositories.
Alternatively, you could reverse engineer the code to create your own trainer.
Could you provide more information about your dataset? Is it publicly available?
